src/uproot/source/fsspec.py

```diff
@@ -33,6 +33,15 @@ def __init__(
     ):
         super().__init__()
         self._coalesce_config = coalesce_config
+
+        # Wrap CERNBox URLs automatically with simplecache::
+        def _maybe_wrap_cernbox(url: str) -> str:
```

Comment:
Better to put this free function outside the init scope

Reply:
thanks! done.